src/consts/Stages.ts

```diff
@@ -33,17 +33,17 @@ type STAGE = STARTSTAGE | MIDSTAGE | ENDSTAGE;
 export const STAGES: ReadonlyArray<STAGE> = [
   {
     key: "Draft",
-    text: "In Draft",
+    text: "RPA Request",
     next: "PackageReview",
     nextEventTitle: "Forward Stage Change: In Draft to Package Review",
     previous: undefined,
     previousEventTitle: undefined,
   },
   {
     key: "PackageReview",
-    text: "Package Review/Approval",
-    next: "Post",
-    nextEventTitle: "Forward Stage Change: Package Review to Posting",
+    text: "Package Review/Concurrence",
+    next: "Recruiting",
+    nextEventTitle: "Forward Stage Change: Package Review to Recruiting",
     previous: "Draft",
     previousEventTitle: "Backward Stage Change: Package Review to In Draft",
```

Comment:
Update the audit log entry
```suggestion
    previousEventTitle: "Backward Stage Change: Package Review to RPA Request",
```